napari/layers/_layer_actions.py

```diff
@@ -93,6 +93,11 @@ def _toggle_visibility(ll: LayerList):
         lay.visible = not lay.visible
 
 
+def _reload(ll: LayerList):
+    for lay in ll.selection:
+        lay.refresh()
```

Comment:
This is not an issue with the PR, but there is an active debate about whether `refresh()` is the right way to trigger a layer reload (as recent as a few minutes ago at the napari community meeting). I'm not sure if we are in a position to commit to `refresh()` being the function that actually triggers what should be a reload.

Reply:
Thanks for the feedback! Is there already some defined alternative of what to call instead of `refresh` to mean a reload? Maybe a new `reload` event/method needs to be created instead of using the `refresh` one then?

Reply:
Just to confuse you even more, there is in fact a public `Layer.events.refresh` event that can be used to update a vispy node. I don't think that's what we want here and probably isn't useful for a GUI-only user. It is not currently emitted in the napari code base, but may once have been and may also be emitted by people developing against napari's public API.

We are also planning to add a public event to trigger a reload/reslice (that may be asynchronous) in https://github.com/napari/napari/pull/5353 . That's currently called `Layer.events.reslice` because we had no better ideas yet. Maybe we should consider waiting for and using that?